app/src/main/java/ca/ubc/best/mint/museandroidapp/vm/TapDetails.java

```diff
@@ -0,0 +1,14 @@
+package ca.ubc.best.mint.museandroidapp.vm;
+
+/** Bundle of information about a single tap. */
+public class TapDetails {
```

Comment:
This style of class might look weird if anyone's taken classes on Java: you're usually taught that you should have private members (e.g. private long reactionTimeMs) then public getters and setters (getReactionTimeMs(), setReactionTimeMs()). 

However, when the value isn't going to change, I've found it common to do this instead: you set it in the constructor, the have the field 'final' (i.e. can't change), as well as public; that way, people can access it as tap.reactionTimeMs rather than tap.getReactionTimeMs().